source/includes/starting-compass-paste-string.rst

```diff
@@ -28,7 +27,20 @@
       .. include:: /includes/fact-hide-connection-string-password.rst
             
       To learn how to format the connection string for a deployment that is not 
-      hosted on |service|, see :manual:`Connection String URI Format </reference/connection-string>`.
+      hosted on |service|, see :manual:`Connection String URI Format
+      </reference/connection-string>`.
+      
+   .. step:: Name your connection (Optional).
```

Comment:
[suggestion] Based on the formatting of other procedures throughout the Compass docs (e.g. [favorite connections](https://www.mongodb.com/docs/compass/current/connect/favorite-connections/), [export pipeline](https://www.mongodb.com/docs/compass/current/agg-pipeline-builder/export-pipeline-to-language/)), we tend to specify the step as being optional _before_ the step text. This is so users can know if the step is optional at a quick glance, rather than having to read through the step, potentially take action, then realize it isn't necessary.
```suggestion
   .. step:: (Optional) Name your connection.
```

Reply:
Fixed!